### PR TITLE
Add Chinese support

### PR DESCRIPTION
**Pull request by johnfercher**
**Description**

**Related Issue**

**Checklist**
> check with "x", if applied to your change

- [ ] All methods associated with structs has ```func (s *struct) method() {}``` name style. 
- [ ] Wrote unit tests for new/changed features. 
- [ ] Updated docs/doc.go 
- [ ] Updated pkg/pdf/example_test.go 
- [ ] Updated README.md 
- [ ] Updated all examples inside internal/examples 
- [ ] New public methods/structs/interfaces has comments upside them explaining they responsibilities 
- [ ] Executed `go fmt github.com/johnfercher/maroto/...` to format all files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,250 +1,45 @@
...
-
-## Examples
-In the [PDFs](internal/examples/pdfs) folder there are the PDFs generated
-using Maroto, and in the [examples](internal/examples) folder there are subfolders
-with the code to generate the PDFs.
-
-![result](internal/assets/images/result.png)
-
-#### Code
+Example
```
Comment (SuddenGunter) [inline README.md:18]: Why the whole readme.md was translated?

File: pkg/pdf/pdf_test.go [hunk 2]
```diff
@@ -3,14 +3,14 @@ package pdf_test
 import (
 	"bytes"
 	"fmt"
-	"github.com/johnfercher/maroto/internal"
-	"github.com/johnfercher/maroto/pkg/color"
+	"github.com/Vale-sail/maroto/internal"
```
Comment (SuddenGunter) [inline pkg/pdf/pdf_test.go:6]: why to change import path? 
current repo would be deprecated?

**Review by SuddenGunter — CHANGES_REQUESTED** [review]

File: pkg/pdf/pdf_test.go (hunk 2 above)
Comment (johnfercher) [reply pkg/pdf/pdf_test.go:6]: no, this is just to merge the utf feature

File: README.md (hunk 1 above)
Comment (johnfercher) [reply README.md:18]: Vale-sail solve a problem with utf, but there are a lot of issues like this.

**Pull request closed without merging**